src/media/media.ts

```diff
@@ -645,7 +646,7 @@ export class SdkMedia extends (EventEmitter as { new(): StrictEventEmitter<Event
    */
   destroy () {
     this.removeAllListeners();
-    window.navigator.mediaDevices.removeEventListener('devicechange', this.handleDeviceChange.bind(this));
+    window.navigator.mediaDevices.removeEventListener('devicechange', this.onDeviceChangeListenerRef);
```

Comment:
Just some appropriate event listener clean up that I missed when setting this up.